rhombus/scribblings/ref-stxobj.scrbl

```diff
@@ -640,7 +638,7 @@ Metadata for a syntax object can include a source location and the raw
 }
 
 @doc(
-  fun Syntax.make_group(terms :: List,
+  fun Syntax.make_group(terms :: Listable,
```

Comment:
Also note that this should to be `.... && NonemptyList`.

Reply:
HF